src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildADesktopExe.cs

```diff
@@ -850,5 +852,154 @@ public void It_allows_TargetFrameworkVersion_to_be_capitalized()
                 .Should()
                 .Pass();
         }
+
+        [WindowsOnlyTheory]
+        [InlineData("true", "true")]
+        [InlineData("true", "false")]
+        [InlineData("false", "true")]
+        [InlineData("false", "false")]
+        public void It_places_package_pdb_and_xml_files_in_output_directory(string enableCopyDebugSymbolFilesFromPackages, string enableDocumentationFilesFromPackages)
+        {
+            var testProject = new TestProject()
+            {
+                Name = "DesktopUsingPackageWithPdbAndXml",
+                TargetFrameworks = ToolsetInfo.CurrentTargetFramework,
+                IsExe = true,
+            };
+
+            testProject.PackageReferences.Add(new TestPackageReference("Microsoft.Build", "17.3.1"));
```

Comment:
Is there a package that has an XML file in the ref folder but not in the lib folder?  That would also be a good test case.

Reply:
I found one "system.diagnostics.debug" Version="4.3.0"

Reply:
done in last commit